src/registrar/admin.py

```diff
@@ -2456,13 +2541,10 @@ def queryset(self, request, queryset):
     fieldsets = (
         (
             None,
-            {"fields": ["name", "state", "expiration_date", "first_ready", "deleted"]},
```

Comment:
I don't know. Will leave to Alysia to answer this one

Reply:
You cannot change a domain name in EPP, since the domain object is a reflection of what is meant to reside in epp, this makes sense that we can't manually change it here. If a domain name change is needed, that actually needs to be it's own record.